dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/LTPAKeyInfoManager.java

```diff
@@ -130,25 +130,25 @@ public synchronized final void prepareLTPAKeyInfo(WsLocationAdmin locService, St
             //load validationKeys
             Iterator<Properties> validationKeysIterator = validationKeys.iterator();
             while (validationKeysIterator.hasNext()) {
-                OffsetDateTime notUseAfterDateOdt = null;
+                OffsetDateTime validUntilDateOdt = null;
                 Properties vKeys = validationKeysIterator.next();
                 String filename = (String) vKeys.get(LTPAConfiguration.CFG_KEY_VALIDATION_FILE_NAME);
                 if (!this.importFileCache.contains(vKeys.get(LTPAConfiguration.CFG_KEY_VALIDATION_FILE_NAME))) {
-                    String notUseAfterDate = ((String) vKeys.get(LTPAConfiguration.CFG_KEY_VALIDATION_NOT_USE_AFTER_DATE));
-                    if (notUseAfterDate != null) {
+                    String validUntilDate = ((String) vKeys.get(LTPAConfiguration.CFG_KEY_VALIDATION_VALID_UNTIL_DATE));
+                    if (validUntilDate != null) {
                         try {
-                            notUseAfterDateOdt = OffsetDateTime.parse(notUseAfterDate);
-                            if (notUseAfterDateOdt != null && isNotUseAfterDate(filename, notUseAfterDateOdt)) {
+                            validUntilDateOdt = OffsetDateTime.parse(validUntilDate);
+                            if (validUntilDateOdt != null && isValidUntilDate(filename, validUntilDateOdt)) {
                                 continue; //Skip this LTPA validationKeys
                             }
                         } catch (Exception e) {
-                            Tr.error(tc, "LTPA_VALIDATION_KEYS_NOT_USE_AFTER_DATE_INVALID_FORMAT", notUseAfterDate, filename);
+                            Tr.error(tc, "LTPA_VALIDATION_KEYS_NOT_USE_AFTER_DATE_INVALID_FORMAT", validUntilDate, filename);
```

Comment:
Change from:
LTPA_VALIDATION_KEYS_NOT_USE_AFTER_DATE_INVALID_FORMAT
To:
LTPA_VALIDATION_KEYS_VALID_UNTIL_DATE_INVALID_FORMAT